src/scripts/stagehand.coffee

```diff
@@ -49,7 +49,8 @@ cancelBooking = (data) ->
 
 module.exports = (robot) ->
   robot.brain.on 'loaded', =>
-    for env in ['staging', 'development', 'production']
+    environments = process.env.STAGEHAND_ENVIRONMENTS.split(',') || [ 'staging', 'development', 'production' ]
```

Comment:
Correct. It'd return false if the property doesn't exist, which would trigger the `||` part of the line.